src/awesome_map/EventSynthesizer.js

```diff
@@ -444,7 +444,7 @@ define(function(require) {
                 var gesture = {
                     deltaX: event.distance.x,
                     deltaY: event.distance.y,
-                    srcEvent: event.source
+                    srcEvent: event.source || event
```

Comment:
We were not passing the source event through for browser events, only for hammer events.